environment_setup/setup_software.sh

```diff
@@ -65,15 +66,6 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-echo "================================================================"
```

Comment:
Why is this removed? Didn't we determine this is necessary for `kcachegrind` to work?

Reply:
Only because the output of `clang` was in a format that required a newer valgrind version. The system one should be fine for anything the system gcc install outputs.